README.md

```diff
@@ -109,15 +109,16 @@ A profile can be identified by any combination of the following:
 
 Identifiers are persisted to local storage so that the SDK can keep track of the current profile.
 
-Profile identifiers and other attributes can be set all at once as a `map`:
+Profile identifiers and other attributes can be set all at once using the `Profile` data class:
 
 ```kotlin
-val profile = Profile(
-    mapOf(
-        ProfileKey.EMAIL to "kermit@example.com",
```

Comment:
In hindsight, this syntax of initializing in all one map is exactly why I made those three keys public. Oh well.

Reply:
Spoke offline, we're going to add a new constructor that can take external_id/email/phone in addition to the map. Not sure we even need to deprecate the original constructor.